ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -38,7 +52,7 @@
 
       Short option: ``-l``.
 
-      Prints the names of mismatched files that were formatted. Prints the names of files that would be formatted when used with ``--check`` mode.
+      Prints the names of mismatched files that were formatted. Prints the names of files that would be formatted when used with ``--check``.
 
    .. cli:option:: --config [key1=val1,key2=val2...]
```

Comment:
Should or must we specify which rustfmt config options there are?

Reply:
hm, not sure... there are so many (78), so perhaps we should consult

Reply:
@pietroalbini what do you think? Maybe we can link to the rustfmt docs and/or include a copy of it? https://github.com/rust-lang/rustfmt/blob/master/Configurations.md

Reply:
I would include a copy of it in a "rustfmt configuration" page or something.

Reply:
Our repo contains it at `src/tools/rustfmt/Configurations.md`. Can and should we tell sphinx to copy it from there?

Reply:
Yeah we should include the file directly from Sphinx.

There is a Sphinx extension called `myst_parser`. It is already installed in our virtual environment (the release notes ext requires it), and you [can enable it as shown in the docs](https://myst-parser.readthedocs.io/en/latest/intro.html#enable-myst-in-sphinx). At that point, including the file *should* just work.

I would do this in a separate PR.

Reply:
Agree to do that in a separate PR.